test/partisan_SUITE.erl

```diff
@@ -38,7 +38,7 @@
 -include_lib("kernel/include/inet.hrl").
 
 -define(APP, partisan).
--define(CT_SLAVES, [rita, sue, bob, jerome]).
+-define(CLIENT_NUMBER, 3).
```

Comment:
It might be useful to not have to use a timer and have a function that we can wait on that returns a stability metric or something.